tsconfig.json

```diff
@@ -13,7 +13,6 @@
     "noUnusedParameters": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
-    "incremental": true,
```

Comment:
Under some circumstances, if a `tsc` build was aborted part-way through, this could cause that build to not run in the future. I ran into that situation while working on this PR. The only change that this should make (other than fixing this problem) is that TS builds _may_ take longer when they are re-run.

Reply:
I believe this will create a problem with the typescript caching schema in CI, we shouldn't change this without more discussion.

Reply:
Reverted in a20a5669a